src/Aspire.Hosting.AWS/Lambda/LambdaEmulatorResource.cs

```diff
@@ -0,0 +1,22 @@
+﻿// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+
+using Aspire.Hosting.ApplicationModel;
+
+namespace Aspire.Hosting.AWS.Lambda;
+
+
+/// <summary>
+/// Resource representing the Lambda Runtime API service emulator.
+/// </summary>
+/// <param name="name">Aspire resource name</param>
+public class LambdaEmulatorResource(string name) : ExecutableResource(name, 
+        "dotnet",
+        Environment.CurrentDirectory
+        )
+{
+    internal void AddCommandLineArguments(IList<object> arguments)
+    {
+        arguments.Add("lambda-test-tool");
```

Comment:
same question on tool name

Reply:
also didn't we change the behavior so that you need to specify the port for the emulator to run? won't this error out as is?